src/ucp/core/ucp_request.h

```diff
@@ -260,6 +260,9 @@ struct ucp_request {
 
                             /* Actual lanes count */
                             uint8_t        lanes_count;
+
+                            /* Memory domains to send remote keys */
+                            ucp_md_map_t   md_map;
```

Comment:
bad placement, we use offset from "new" union part in the same flow, see ucp_rndv_req_send_rtr